.goreleaser.yaml

```diff
@@ -38,7 +38,8 @@ builds:
       post:
         - output: true
           # checks that we've successfully statically built this executable
-          cmd: ldd {{ .Path }} 2>&1 | grep 'not a dynamic executable'
+          cmd: powershell -Command "if ( -Not (ldd {{ .Path }} 2>&1 | findstr 'not a dynamic executable')) { exit 1 }"
```

Comment:
How about we always print the output of ldd, so that we're able to see what the output was, in case we have similar problems of misdetecting this?